chore: polish kb31-40 files (#1004)
keymanapp/help.keyman.com: knowledge-base/kb0034.md

# You receive a "Unable to copy file (filename) to (folder) - you may have insufficient privileges, or you may have run out of disk space" message when installing a keyboard

### **NOTE**: This archived documentation has not been updated recently and may contain information that is no longer relevant

## Symptoms

When you install a keyboard or a package for Keyman Desktop 7.0, you may receive one of the following error messages:

> Unable to copy file _(filename)_ for the package to _(path)_- you may have insufficient privileges, or you may have run out of disk space.
> 
> Unable to copy file _(filename)_to _(path)_- you may have insufficient privileges, or you may have run out of disk space.

## Cause
This error can occur when a corrupted keyboard installation has been left on the computer.

## Resolution
To resolve this issue, you must delete the corrupted keyboard from your computer. Take a note the _path_ given in the error message.

> - Click **Start** menu, **Run** (on Windows Vista, hold Windows key and press R)
> - Enter the path given in the error message and press OK.
> - In Explorer, navigate to the parent folder and delete the folder you originally navigated to.

Author: ```suggestion
1. Click **Start** menu, **Run** (on Windows Vista, hold Windows key and press R)
2. Enter the path given in the error message and press OK.
3. In Explorer, navigate to the parent folder and delete the folder you originally navigated to.

```

# You receive a "Unable to copy file (filename) to (folder) - you may have insufficient privileges, or you may have run out of disk space" message when installing a keyboard

### **NOTE**: This archived documentation has not been updated recently and may contain information that is no longer relevant

## Symptoms

When you install a keyboard or a package for Keyman Desktop 7.0, you may receive one of the following error messages:

> <div markdown="1" style='color:red'>
> 
>         Unable to copy file _(filename)_ for the package to _(path)_- you may have insufficient privileges, or you may have run out of disk space.
> 
>         Unable to copy file _(filename)_to _(path)_- you may have insufficient privileges, or you may have run out of disk space.
> 
> </div>

Reviewer: @mcdurdin - is there a better way to style red text in a code block?

```suggestion
<code><span style="color: red">
Unable to copy file _(filename)_ for the package to _(path)_- you may have insufficient privileges, or you may have run out of disk space.<br>
Unable to copy file _(filename)_to _(path)_- you may have insufficient privileges, or you may have run out of disk space.
</code>
```

Reviewer 2: No, I suggest we don't maintain the colour. Instead use normal markdown.

## Cause
This error can occur when a corrupted keyboard installation has been left on the computer.

## Resolution

> To resolve this issue, you must delete the corrupted keyboard from your computer.  Take a note the __path_given in the error message.

Reviewer: Needs a space to italicize "path"

```suggestion
To resolve this issue, you must delete the corrupted keyboard from your computer. Take a note the _path_ given in the error message.
```
